In the `lint` workflow, only normal and `dev` dependencies are being installed. `geopandas` is a doc dependency so far and hence, mypy can't find it. As we now use it in this example which is also a test, geopandas should be moved from `doc` dependencies to `dev` in pyproject.toml. As geopandas is not a typed package as far as I know, you'll first get another error:
```
tests/examples_methods_syntax/interval_selection_map_quakes.py:11: error: Skipping analyzing "geopandas": module is installed, but missing library stubs or py.typed marker  [import-untyped]
```
which you can solve by adding `geopandas.*` to the modules under `[[tool.mypy.overrides]]` in `pyproject.toml`.

These two steps should make mypy happy :)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -72,7 +72,8 @@ dev = [
     "types-setuptools",
     "pyarrow>=11",
     "vegafusion[embed]>=1.4.0",
-    "anywidget"
+    "anywidget",
+    "geopandas",
 ]
 doc = [
     "sphinx",
@@ -82,7 +83,6 @@ doc = [
     "numpydoc",
     "pillow>=9,<10",
     "pydata-sphinx-theme>=0.14.1",
-    "geopandas",
     "myst-parser",
     "sphinx_copybutton",
     "sphinx-design",
@@ -216,6 +216,7 @@ module = [
     "yaml.*",
     "vl_convert.*",
     "pandas.lib.*",
+    "geopandas.*",
     "nbformat.*",
     "ipykernel.*",
     "m2r.*",

--- a/tests/examples_arguments_syntax/interval_selection_map_quakes.py
+++ b/tests/examples_arguments_syntax/interval_selection_map_quakes.py
@@ -16,7 +16,7 @@ gdf_world = gpd.read_file(data.world_110m.url, driver="TopoJSON")
 
 # defintion for interactive brush
 brush = alt.selection_interval(
-    encodings=['longitude'], 
+    encodings=["longitude"], 
     empty=False, 
     value={"longitude": [-50, -110]}
 )
@@ -38,7 +38,7 @@ quakes = alt.Chart(gdf_quakies).transform_calculate(
 ).mark_circle(opacity=0.35, tooltip=True).encode(
     longitude="lon:Q",
     latitude="lat:Q",
-    color=alt.condition(brush, alt.value('goldenrod'), alt.value('steelblue')),
+    color=alt.condition(brush, alt.value("goldenrod"), alt.value("steelblue")),
     size=alt.Size("mag:Q", scale=alt.Scale(type="pow", range=[1, 1000], domain=[0, 7], exponent=4)),
 ).add_params(brush)
 
@@ -47,13 +47,13 @@ left_map = alt.layer(sphere, world, quakes).project(type="mercator")
 
 # histogram of binned earthquakes
 bars = alt.Chart(gdf_quakies).mark_bar().encode(
-    x=alt.X('mag:Q').bin(extent=[0,7]),
-    y='count(mag):Q',
-    color=alt.value('steelblue')
+    x=alt.X("mag:Q").bin(extent=[0,7]),
+    y="count(mag):Q",
+    color=alt.value("steelblue")
 )
 
 # filtered earthquakes
-bars_overlay = bars.encode(color=alt.value('goldenrod')).transform_filter(brush)
+bars_overlay = bars.encode(color=alt.value("goldenrod")).transform_filter(brush)
 
 # combine layers for histogram
 right_bars = alt.layer(bars, bars_overlay)

--- a/tests/examples_methods_syntax/interval_selection_map_quakes.py
+++ b/tests/examples_methods_syntax/interval_selection_map_quakes.py
@@ -16,7 +16,7 @@ gdf_world = gpd.read_file(data.world_110m.url, driver="TopoJSON")
 
 # defintion for interactive brush
 brush = alt.selection_interval(
-    encodings=['longitude'], 
+    encodings=["longitude"], 
     empty=False, 
     value={"longitude": [-50, -110]}
 )
@@ -38,7 +38,7 @@ quakes = alt.Chart(gdf_quakies).transform_calculate(
 ).mark_circle(opacity=0.35, tooltip=True).encode(
     longitude="lon:Q",
     latitude="lat:Q",
-    color=alt.condition(brush, alt.value('goldenrod'), alt.value('steelblue')),
+    color=alt.condition(brush, alt.value("goldenrod"), alt.value("steelblue")),
     size=alt.Size("mag:Q").scale(type="pow", range=[1, 1000], domain=[0, 7], exponent=4),
 ).add_params(brush)
 
@@ -47,13 +47,13 @@ left_map = alt.layer(sphere, world, quakes).project(type="mercator")
 
 # histogram of binned earthquakes
 bars = alt.Chart(gdf_quakies).mark_bar().encode(
-    x=alt.X('mag:Q').bin(extent=[0,7]),
-    y='count(mag):Q',
-    color=alt.value('steelblue')
+    x=alt.X("mag:Q").bin(extent=[0,7]),
+    y="count(mag):Q",
+    color=alt.value("steelblue")
 )
 
 # filtered earthquakes
-bars_overlay = bars.encode(color=alt.value('goldenrod')).transform_filter(brush)
+bars_overlay = bars.encode(color=alt.value("goldenrod")).transform_filter(brush)
 
 # combine layers for histogram
 right_bars = alt.layer(bars, bars_overlay)

